test/HybridCustody_tests.cdc

```diff
@@ -2,11 +2,13 @@ import Test
 
 pub var accounts: {String: Test.Account} = {}
 pub var blockchain = Test.newEmulatorBlockchain()
+pub let flowTokenAccount: Address = 0x0ae53cb6e3f42a79
```

Comment:
Seems like this and the other variable aren't being used?

Reply:
Yeah, remnant from when I was trying to write a case for the one that was failing. I'll clean that up now.